components/Client.js

```diff
@@ -1,44 +1,37 @@
 function Client() {
   return (
     <section className="text-white">
-      <div className="max-w-screen-xl px-4 py-16 mx-auto sm:px-6 lg:px-8 sm:py-24">
+      <div className="max-w-screen-xl px-4 py-32 mx-auto sm:px-6 lg:px-8 sm:py-40">
         <div className="max-w-xl mx-auto text-center">
-          <h2 className="text-4xl font-bold tracking-tight sm:text-5xl">
+          <h2 className="text-4xl font-inter-400 font-semibold tracking-tight sm:text-5xl">
```

Comment:
@supersver 
It is bad practice to use fonts like this. Use font-inter globally, then just use font-bold, font-semibold, etc. for font weight. You have also used font-inter-400 and font-semibold. Doesn't make sense